internal/kms/kms.go

```diff
@@ -61,6 +79,41 @@ type KMS interface {
 	// by the key ID. The contexts must match the context value
 	// used to generate the ciphertexts.
 	DecryptAll(ctx context.Context, keyID string, ciphertext [][]byte, context []Context) ([][]byte, error)
+
+	// DescribePolicy describes a policy by returning its metadata.
```

Comment:
For these, that are quite specific, would it make sense to add these as separate interfaces?

Bloating the `KMS` interface reduces the usability and forces stubs for other implementations .

Go Proverb: [The bigger the interface, the weaker the abstraction](https://www.youtube.com/watch?v=PAAkCSZUG1c&t=5m17s)

Instead add `kms.KeyManager`, `kms.Policy` and `kms.Identity` interfaces could be added and the caller can assert them on the KMS for whatever operation they need.

Usage would look like this:
```
var globalKMS kms.KMS

func listPolicy(...) error {
    if k, ok := globalKMS.(kms.Policy); ok {
      k.DescribePolicy(...)
    }
    return errors.New("kms: describe policy is not supported")
}
```

... or alternatively:
```
var globalKMSPolicy kms.Policy

func listPolicy(...) error {
    if globalKMSPolicy == nil {
      return errors.New("kms: describe policy is not supported")
    }
    globalKMSPolicy.DescribePolicy(...)
}
```
... or you could have a dummy implementation of `kms.Policy` that just returns errors.

There are plenty of options where you don't need to pollute the interface.